infra/tempo/config/tempo.yaml

```
@@ -0,0 +1,62 @@
+metrics_generator_enabled: true
+search_enabled: true
+
+server:
+  http_listen_port: 3200
```

```suggestion
  http_listen_port: ${TEMPO_PORT}
```

It's better to have those in variables.